### ISSUE TITLE
EVERYONE: Please consider sponsoring Ian

### ISSUE THREAD

**Issue by Laterallus** [ianmarmour/nvidia-clerk#138]
Hey everyone,

I'm not super familar with GitGub and couldn't find anywhere to post this, so I'm posting it here. I know this is meant for surfacing issues, and I'm sorry if posting this here is a problem (remove it if so), but I want my post to be seen by others.

Ian built a great bot and is SUPER responsive to issues that are raised. He could have kept this to himself, scored a card, and moved on. Instead, he decided to share this with the gaming community and I have great respect for that. I hope others will follow my lead and show their gratitude in a way that counts by being a sponsor. 

THANK YOU IAN!

**Comment by redstone2019** [comment]
https://github.com/sponsors/ianmarmour

**Comment by hymer08** [comment]
I agree, It's really awesome how much effort he has put in this project. He even got a card himself days ago but is still keeping this running for us.

**Comment by dexmar** [comment]
Much appreciated (and now sponsored)!

The discord link is expired, can anyone drop one here?

**Comment by redstone2019** [comment]
it is?

**Comment by redstone2019** [comment]
click on chat on the homepage

**Comment by dexmar** [comment]
Funny! I was clicking on the link lower on the main page. Got it now, thanks!

**Comment by ianmarmour** [comment]
I really appreciate it guys, if you don't want to do a reoccurring donation to me please also consider sponsoring code.org instead for a one off or re-occurring donation.  I love the work that they're doing and want to see more STEM funding globally for children <3

**Comment by dexmar** [comment]
Happy to do it! I'm going to be at this for a bit as I want to get a card for myself and my 3 kids so as long as you are updating  I in tend to keep helping to fund.

Again, thanks for all of the effort you and others have put into this!

**Comment by penguinologist** [comment]
I've donated to code.org before as well as helped organize a few of their local hackdays. They're awesome. Thanks for the work @ianmarmour. Got me motivated to help out in the open-source community.

**Comment by emike09** [comment]
I just snagged an RTX 3080 with this app, thanks Ian for developing it! Can confirm it works as expected. Now to play the waiting game with processing and shipping!

**Comment by bmorera8** [comment]
I said this on the Discord, and I'm saying it again here for those that didn't see it:

This bot took time and effort to build and probably even more to maintain, yet it was given to us for FREE. 

If you got a card, please consider sponsoring Ian on Github, or if you want to make a one-time payment, Ian is asking that you make a donation to [code.org.](https://donate.code.org/campaign/support-computer-science-education/c172233) The mere fact that he prefers to divert payment to a charity should tell you everything.

**Comment by Getyourcardnow** [comment]
ianmarmour, thank you! I missed the window earlier today but hoping to grab a 3090 in the next few days. Really appreciate your effort
